examples/summarize_to_notion/docker-compose.yml

```diff
@@ -15,7 +15,8 @@ services:
 
   taotie:
     build:
-      context: ${PWD}
+      # Should be a relative path to the Compose file’s parent folder (${PWD} doesn't work on Ubuntu)
+      context: ../../
```

Comment:
Does this mean we should run the command in the folder summarize_to_notion? We may want to check whether Dockerfile will work as expected in this way.